### PR TITLE
feat: added animate util classes and badge classes

### PR DESCRIPTION
**Pull request by VedanthB**
- added animation util classes
- added badge classes

https://user-images.githubusercontent.com/75097551/151694123-729c9231-61c4-43fa-b7ef-7473c13f22c0.mov

### CODE REVIEW

**Comment by rkapoor10** [comment]
great badges!

File: index.html [hunk 1]
```diff
@@ -21,44 +21,44 @@
...
       <img
-        src="https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale"
+        src="components/images/avatar-img.jpeg"
         alt="avatar-img"
         srcset="
-          https://thumbor.forbes.com/thumbor/711x711/https://specials-images.forbesimg.com/imageserve/613df8e8d679a21b766a1636/bigbun-2/960x0.jpg?fit=scale
+          components/images/avatar-img.jpeg
         "
         class="img-round img-responsive"
       />
```
Comment (rkapoor10) [inline index.html:64]: you may add loading="lazy" as these badges need not load immediately with the doc site.

**Review by rkapoor10 — COMMENTED** [review]
Good to go

File: index.html (hunk 1 above)
Comment (VedanthB) [reply index.html:64]: okay will make the change

**Comment by sharathnair9999** [comment]
looks nice vedanth. good to go

**Comment by SukanyaSen530** [comment]
Looks Good :)

**Review by SukanyaSen530 — APPROVED** [review]

**Pull request merged**